extensions/data-transfer/portability-data-transfer-daybook/src/test/java/org/datatransferproject/transfer/daybook/social/DaybookPostsImporterTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2022 The Data-Portability Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.datatransferproject.transfer.daybook.social;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.junit.Assert.*;
```

Comment:
Generally good code style is to avoid wildcard imports https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
I'd recommend using https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml for code style on intellij and the google-java-format plugin https://plugins.jetbrains.com/plugin/8527-google-java-format

Reply:
Thank you for pointing out! I had the plugin, but didn't realise I also have to apply the config in the settings.